docs/UserGuide.md

```diff
@@ -284,13 +283,14 @@ logout
 
 </box>
 
-## Adding your first Delivery
+## Registering and creating your first Delivery
 
 Hopefully at this point, you have a good understanding of the basics of HomeBoss. Let's get you started on HomeBoss
-by adding your first Delivery
+by creating your first Delivery
 
 1. First register for HomeBoss using the `register`{.swift} command, more details on the command can be found
-   [here](#register)
+   [here](#register). So, for example, type `register --user Gabriel --password GabrielIsGreat --confirmPass 
+   GabrielIsGreat --secretQn First Pet Name? --answer Koko`{.swift} into the Command Box and hit enter.
```

Comment:
Should the more details be placed at the end rather than the middle?

Reply:
sure